### PR TITLE
New annotation pop up rearrangement done.

### PR DESCRIPTION
**Pull request by kthorat-prog**

### CODE REVIEW

File: src/main/cliapp/src/containers/diseaseAnnotationsPage/NewAnnotationForm.js [hunk 1]
```diff
@@ -30,7 +31,7 @@ import { AssertedGenesAdditionalFieldData } from "../../components/AssertedGenes
 import { EvidenceCodesAdditionalFieldData } from "../../components/EvidenceCodesAdditionalFieldData";
 import { WithAdditionalFieldData } from "../../components/WithAdditionalFieldData";
 import { GeneticModifierAdditionalFieldData } from "../../components/GeneticModifierAdditionalFieldData";
-import ErrorBoundary from "../../components/Error/ErrorBoundary";
```
Comment (adamgibs) [inline src/main/cliapp/src/containers/diseaseAnnotationsPage/NewAnnotationForm.js:33]: The ErrorBoundary component would still need to be in place in order to have Error Boundaries appear in the form if there is an error. For example, if one of the form's autocompletes attempts to display a field that isn't in the api response.

Comment (kthorat-prog) [reply src/main/cliapp/src/containers/diseaseAnnotationsPage/NewAnnotationForm.js:33]: I will rebase it to alpha and check for these changes.

Comment (kthorat-prog) [reply src/main/cliapp/src/containers/diseaseAnnotationsPage/NewAnnotationForm.js:33]: Please check for errorboundry component.

Comment (adamgibs) [reply src/main/cliapp/src/containers/diseaseAnnotationsPage/NewAnnotationForm.js:33]: @kthorat-prog Looks good to me!

**Review by oblodgett — APPROVED** [review]

**Comment by oblodgett** [comment]
@kthorat-prog might want to clean these up:
```
[eslint] 
src/containers/diseaseAnnotationsPage/NewAnnotationForm.js
  Line 7:10:   'MultiSelect' is defined but never used                 no-unused-vars
  Line 35:10:  'ConfirmButton' is defined but never used               no-unused-vars
  Line [54](https://github.com/alliance-genome/agr_curation/actions/runs/4789920056/jobs/8518358077?pr=990#step:7:55):9:   'selectedFields' is assigned a value but never used     no-unused-vars
  Line 54:25:  'setSelectedFields' is assigned a value but never used  no-unused-vars
  Line 410:8:  'defaultFields' is assigned a value but never used      no-unused-vars
```

**Pull request merged**